rules/integrations/okta/credential_access_user_impersonation_access.toml

```diff
@@ -0,0 +1,44 @@
+[metadata]
+creation_date = "2022/03/22"
+maturity = "production"
+updated_date = "2022/03/22"
+integration = "okta"
+
+[rule]
+author = ["Elastic"]
+description = """
+A user has initiated a session impersonation granting them access to our environment with the permissions of the user
+they are impersonating. This would likely indicate Okta administrative access and should only ever occur if requested
+and expected.
```

Comment:
```suggestion
A user has initiated a session impersonation granting them access to the environment with the permissions of the user
they are impersonating. This would likely indicate Okta administrative access and should only ever occur if requested
and expected.
```